docs/builder-general-resources.md

```diff
@@ -6,12 +6,21 @@ sidebar_position: 3
 
 ## General Resources
 
+[ETH Chicago Hackathon Home](https://taikai.network/ethchicago/hackathons/ETHChicagoHackathon2023)
+
 ​[ETHGlobal Interactive Guides](https://ethglobal.com/guides)​ - curated list of useful tools, resources, and open-source projects that will help you get started with web3, by ETHGlobal
 
 ​[Ethereum.org's Getting Starting Page](https://ethereum.org/en/developers/learning-tools/) - list of basic guides to get started
 
 [Ethereum.org's Developer Resources](https://ethereum.org/en/developers/)​ - Guides, IDEs, tools, libraries
 
+## Hack a Hackathon Support
+
+[The Hackathon Method by OpenUX](https://mirror.xyz/openux.eth/Z9CkF5Np4usTAt393Snq-fLN6BfZQkj-5_o6dy-XGoE) - Data-driven advice for hackers participating in hackathons. 
+(A long piece, but we believe there's value for hackers of every experience level. Recommend reading before the event.)
+
+[Cambridge Hacks: Hackathon Prep Guide](https://mirror.xyz/openux.eth/Z9CkF5Np4usTAt393Snq-fLN6BfZQkj-5_o6dy-XGoE) - Thorough documentation to make the most of your hackathon experience.
```

Comment:
can we extract the hackathon prep guide into its own page? instead of having it here ( it's not for cambridge hacks) they repurposed it for us.